examples/language-sdk-instrumentation/golang-push/rideshare/rideshare/rideshare.go

```diff
@@ -358,12 +326,22 @@ func MeterProvider(c Config) (*sdkmetric.MeterProvider, error) {
 	}
 
 	ctx := context.Background()
-	exp, err := otlpmetrichttp.New(ctx)
+	opts := []otlpmetrichttp.Option{otlpmetrichttp.WithEndpoint(c.OTLPUrl)}
```

Comment:
I've found another issue. I think this is the culprit causing this,  since it overrides the `OTEL_EXPORTER_OTLP_METRICS_ENDPOINT` environment variable configuration:
```
2025/04/23 07:47:43 failed to upload metrics: Post "http:///api/v1/otlp/v1/metrics": http: no Host in request URL
```

After the changes, try to run the example to see if the otlp metrics added in this PR https://github.com/grafana/pyroscope/pull/3905 are still available.

Reply:
Still seeing the error above when running the example with `docker-compose up --build`. Could you please another look?

Reply:
Still seeing the error above when running the example with `docker-compose up --build`. Could you please another look?

Reply:
I think it should be working properly now, please take another look @marcsanmi